mysql-test/suite/tokudb.rpl/t/rpl_tokudb_stm_log-master.opt

```diff
@@ -0,0 +1,2 @@
+--skip-external-locking
+--default-storage-engine=MyISAM
```

Comment:
Probably not, not even sure where this file came from as there isn't a matching file in the rpl suite where this was supposedly copied from.